test/serializer/optimized-functions/DeadObjectAssign23.js

```diff
@@ -0,0 +1,38 @@
+// Copies of _\$C\(:2
+// Copies of var _\$C = _\$B.assign;:1
+// inline expressions
+
+// _$C is the variable for Object.assign. See DeadObjectAssign4.js for
+// a larger explanation.
+
+function fn2(obj, x) {
+  return Object.assign({}, x, { a: 1 });
+}
+
+function fn3(obj, x) {
+  return Object.assign({}, x, { b: 1 });
+}
+
+function fn(obj, x) {
+  var a = fn2(obj, x);
```

Comment:
`fn2` and `fn3` are just getting executed and could be inlined, or are they needed for the regression test?